src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -1,10 +1,15 @@
 package de.retest.recheck;
 
+import java.io.File;
+
+import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.ignore.RecheckIgnoreUtil;
+import de.retest.recheck.review.GlobalIgnoreApplier;
 import lombok.Builder;
 import lombok.Getter;
-import lombok.experimental.SuperBuilder;
 
-@SuperBuilder
+@Builder
```

Comment:
IIRC this breaks compatability with `RecheckWebOptions`.

Reply:
As discussed...